### PR TITLE
Debugged.

### LINKED ISSUE #55: App Crashing if setCornerRadius is written before setData method.
I want to work on this.

### PR DESCRIPTION
**Pull request by Alaska18**
Fixes #55 
Debugged

### CODE REVIEW

**Comment by ShivangiSingh17** [comment]
Which issue are you addressing @Alaska18 ?

**Comment by Alaska18** [comment]
App crashing if setCornerRadius method is written before set data.

**Comment by ShivangiSingh17** [comment]
Please try giving a proper description while creating the pull request. Follow the contributing guidelines properly.

File: library/src/main/java/in/krharsh17/barview/BarView.java [hunk 1]
```diff
@@ -24,6 +24,7 @@
     private OnBarClickListener onBarClickListener;
     private List<BarGroup> barGroups;
     private List<BarModel> data;
+    private boolean isDataPopulated;
```
Comment (ShivangiSingh17) [inline library/src/main/java/in/krharsh17/barview/BarView.java:27]: Does adding this really solve the app crash?

**Comment by Alaska18** [comment]
The App was crashing because of the null pointer reference to populateBarView();
Boolean variable checks if the data is populated or not and depending on the condition it executes it. If set corner radius is written before setData then this variable will be false and it will not execute populateBarView();.

**Comment by ShivangiSingh17** [comment]
@Alaska18 Did you try running the code you have suggested?
I understood what you tried doing but for me the app is still crashing.

**Comment by Alaska18** [comment]
Yeah. I tried running it. It is working. 
There is another reason of App getting crashed here. That is in the typeface of label and value. It could not find the specified path mentioned in the code to assign a font to them. If you comment those lines and try running it , it should work.

**Review by ShivangiSingh17 — APPROVED** [review]

**Comment by ShivangiSingh17** [comment]
Okay

**Pull request merged**